comp_sci_gender_bias/getters/dfe_school_data.py

```diff
@@ -0,0 +1,98 @@
+from comp_sci_gender_bias import PROJECT_DIR
+import pandas as pd
+import pathlib
+from comp_sci_gender_bias.utils.process_pandas import cols_replace_space_and_lowercase
+
+
+DFE_DIR = PROJECT_DIR / "inputs/data/dfe_school_info/"
+DFE_2021_DIR = DFE_DIR / "2020-2021/"
+DFE_2019_DIR = DFE_DIR / "2018-2019/"
```

Comment:
I have a small preference for using `os.path.join` as it makes these long path joins a bit cleaner and is operating system independent.

Reply:
I like using path objects as I find them easier to read and they have extra functionality built in.